src/code_80057C60.c

```diff
@@ -2784,31 +2784,31 @@ void func_8005D290(void) {
     D_8018D4C8 = dma_textures(gTextureLightningBolt1, 0x800, 0x800);
 }
 
-void func_8005D6C0(Player* player) {
+void reset_player_particle_pool(Player* player) {
```

Comment:
What does pool mean?

reset_player_particle_emitter ?

This looks more like init_particle_emitter